weblab/templates/experiments/experiment_tasks.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+{% load experiments %}
+{% load staticfiles %}
+
+{% block title %}Tasks - {% endblock title %}
+
+{% block content %}
+
+    <div id="taskHeader">
+        <h2>List of active tasks</h2>
+        <a id="taskRefreshPage"><img src="{% static 'img/refresh.png' %}" alt="Refresh tasks"
+                                     title="Refresh tasks"/></a>
+    </div>
+
+    {% if  object_list %}
+        <p>
+            [<small><a id="cancelRunningExperimentsAll">select all</a></small>]
+            [<small><a id="cancelRunningExperimentsNone">select none</a></small>]
+        </p>
+    {% endif %}
+
+    <form action="" method="post">
+        {% csrf_token %}
+
+        {% if  object_list %}
+            <p>
+                <button type="submit">Cancel selected experiments</button>
+            </p>
+        {% endif %}
+        <div id="experimentversionlist">
+            {% if not object_list %}
+                <p class="failed">You have no queued or running tasks.</p>
+            {% endif %}
+
+            {% for experiment in object_list|dictsort:"name" %}
+                {{ experiment.name }}
+                <div class="suppl">
+                    <small>Experiment created
+                        <time>{{ experiment.created_at }}</time>
+                        by <em>{{ experiment.author.full_name }}</em>.
+                    </small>
+                </div>
+                <ul id="taskList">
+                    {% for version in experiment.versions.all|dictsortreversed:"created_at" %}
```

Comment:
Will need to check that previous completed versions of an experiment don't also show up here.

Reply:
I have done some tests and it seems to filter by currently running experiment

Reply:
now covered by unit tests - thanks